### PR TITLE
fix(home page): #MED-272 new display on home page

### PR DESCRIPTION
**Pull request by niko-amosse**
## Describe your changes

image

change home display for different cases of empty list (favorite, manual, etc) and add an empty state

## Checklist tests

check the display with different combinaison of empty list
check the responsive

## Issue ticket number and link

[MED-272](https://jira.support-ent.fr/browse/MED-272)

## Checklist before requesting a review (magic string, indentation, comment/documentation...)

- [ ] I have detailed the tests to do in my feature/fix in order to prevent consequents regressions (must specify in **Checklist tests**)
- [X] I have performed a self-review of my code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation (API Doc etc...) - (must specify in **Description** for target version)
- [ ] If it is a consequent feature, I have added thorough tests.
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] Any dependent changes have been added to this project (must specify in **Description**)

### CODE REVIEW

File: frontend/src/hooks/useTextbook.ts [hunk 1]
```diff
@@ -2,28 +2,44 @@ import { useEffect, useState } from "react";
...
-      let textbookData: Textbook[] = textbook?.data?.textbooks ?? [];
-      textbookData = textbookData.map((textbook: Textbook) => ({
+      let garData: Textbook[] = textbook?.data?.textbooks ?? [];
+      garData = garData.map((textbook: Textbook) => ({
         ...textbook,
         favorite: favorites.some((fav: Favorite) => fav.id === textbook.id),
       }));
+      const textbookData = garData.filter(
+        (textbook: Textbook) =>
+          textbook.document_types?.includes("livre numérique") ?? false,
```
Comment (JinDouang) [inline frontend/src/hooks/useTextbook.ts:26]: je peux pas accepter ça

de ce que je comprends, vous reprenez une fonctionnalité déjà existante en angularjs, montrez le code/commit

Si, une règle de gestion s'est rajouté, je veux les specs dans le ticket montrant explicitement que "livre numérique", c'est "manuel"

File: frontend/src/hooks/useTextbook.ts [hunk 2]
```diff
@@ -2,28 +2,44 @@ import { useEffect, useState } from "react";
...
+      );
+      const externalResourceData = garData.filter(
+        (externalResource: ExternalResource) =>
+          !externalResource.document_types?.includes("livre numérique") ??
```
Comment (JinDouang) [inline frontend/src/hooks/useTextbook.ts:30]: idem

**Review by JinDouang — CHANGES_REQUESTED** [review]

File: frontend/src/components/list-card/ListCard.scss [hunk 3]
```diff
@@ -16,6 +16,31 @@
     background-color: $med-white;
     border-radius: 5px;
     box-shadow: 0px 2px 3px 0px #0000001f;
+    @media (min-width: 1280px) {
+      height: 680px;
```
Comment (alicedraillard) [inline frontend/src/components/list-card/ListCard.scss:20]: Pour toutes les valeurs en dur : OK pour le responsive ?

File: frontend/src/styles/page/home.scss [hunk 4]
```diff
@@ -24,6 +24,22 @@ main {
...
+        width: 100%;
+      }
+      display: flex;
+      justify-content: center;
+      .empty-state {
+        display: flex;
+        flex-direction: column;
+        align-items: center;
+        justify-content: center;
+        gap: 20px;
```
Comment (alicedraillard) [inline frontend/src/styles/page/home.scss:37]: idem pr les valeurs en dur : ok responsive ?

File: frontend/src/components/list-card/ListCard.scss (hunk 3 above)
Comment (niko-amosse) [reply frontend/src/components/list-card/ListCard.scss:20]: oui c'est testé

File: frontend/src/styles/page/home.scss (hunk 4 above)
Comment (niko-amosse) [reply frontend/src/styles/page/home.scss:37]: idem testé

**Pull request merged**